packages/common/src/hdwallets.ts

```diff
@@ -71,11 +71,11 @@ export const bigNumberFieldsToStrings = (
     (transaction, txnProperty) => ({
       ...transaction,
       ...((transaction[txnProperty as keyof TransactionRequest] as BigNumber)
-        .toHexString
+        instanceof BigNumber
```

Comment:
I did some testing and since these values are coming from the populate transaction method in Ethers, it will be the Ethers instance of BigNumber which use BN.js, so the `instanceof` check will not work. So I think we can actually revert this to the original code.